packit_service/models.py

```diff
@@ -117,6 +118,7 @@ class GitProjectModel(Base):
     branches = relationship("GitBranchModel", back_populates="project")
     releases = relationship("ProjectReleaseModel", back_populates="project")
     issues = relationship("IssueModel", back_populates="project")
+    merge_requests = relationship("MergeRequestModel", back_populates="project")
```

Comment:
I would also reuse the `PullRequestModel`. Also in ogr, we use `pull-request` for all git-forges. (Maybe put a note to the docstring that it can be merge-request as well.)